.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "tests/test_helper/bats-support"]
+	path = tests/test_helper/bats-support
+	url = https://github.com/bats-core/bats-support.git
```

Comment:
While I at first glance thought the submodule approach was a bit odd for a test dependency, I do prefer it over having to pollute the system running the test with `npm install`, so ++

I also see that these are available as packages in recent Ubuntu releases, so come next LTS we may no longer need to do this:
https://launchpad.net/ubuntu/+source/bats-assert
https://launchpad.net/ubuntu/+source/bats-support